addons/website_sale_slides/models/slide_channel.py

```diff
@@ -75,17 +76,17 @@ def action_view_sales(self):
         action['domain'] = [('product_id', 'in', self.product_id.ids)]
         return action
 
-    def _filter_add_members(self, target_partners):
+    def _filter_add_members(self, target_partners, raise_on_access=False):
         """ Overridden to add 'payment' channels to the filtered channels. People
         that can write on payment-based channels can add members. """
-        result = super(Channel, self)._filter_add_members(target_partners)
+        result = super(Channel, self)._filter_add_members(target_partners, raise_on_access=raise_on_access)
         on_payment = self.filtered(lambda channel: channel.enroll == 'payment')
         if on_payment:
             try:
                 on_payment.check_access_rights('write')
                 on_payment.check_access_rule('write')
-            except:
-                pass
+            except AccessError:
+                raise AccessError(_('You are not allowed to add members to this course. Please contact the course responsible or an administrator.'))
```

Comment:
should check if raise on access is true :)

Reply:
My bad, thanks!